templates/pages/checkout.html

```diff
@@ -1 +1,35 @@
-brand.html
+<!doctype html>
+<html>
+<head>
+    <base href="/">
+    <meta charset="UTF-8"/>
+    <meta name="keywords" content="checkout"/>
+    <meta name="description" content="Bigcommerce checkout"/>
+    <meta name="robots" content="noindex, nofollow">
+    <meta name="viewport" content="width=device-width">
+    <link rel="SHORTCUT ICON" href="{{ FaviconUrl }}" />
+    {{{ open_sans_stylesheet }}}
+    {{{ ng_checkout_stylesheet }}}
+    {{{ stylesheet '/assets/css/optimized-checkout.css' }}}
+    {{ getFontsCollection }}
+    {{ OptimizedCheckoutWebdavCss }}
+</head>
+    <body>
+<header class="checkoutHeader optimizedCheckout-header">
+    <div class="checkoutHeader-content">
+        <h1 class="is-srOnly" translate>Checkout</h1>
+        <h2 class="checkoutHeader-heading">
+            <a class="checkoutHeader-link" href="/">
+                {{#if HeaderImage }}
+			<img alt="{{ StoreName }}" class="checkoutHeader-logo" id="logoImage" src="{{ HeaderImage }}"/>
```

Comment:
Some of these Stencil objects seem to be duplicates of existing Stencil objects, so we probably don't need them, except maybe the store logo (as we have a separate field to upload a store logo specifically for checkout). Have a look at `templates/components/common/store-logo.html` and you will see what I mean.